app/javascript/packs/onboardingRedirectCheck.jsx

```diff
@@ -39,7 +39,11 @@ document.ready.then(
 window.InstantClick.on('change', () => {
   getUserDataAndCsrfToken()
     .then(({ currentUser }) => {
-      if (redirectableLocation() && !onboardingSkippable(currentUser)) {
+      if (
+        redirectableLocation() &&
+        localStorage.getItem('shouldRedirectToOnboarding') === null &&
```

Comment:
If the user has not completed the CoC/T&C page yet, then the local storage key `shouldRedirectToOnboarding` would not be set yet and therefore would equal `null`.